### PR TITLE
feat: add events command

### PR DESCRIPTION
**Pull request by NathanInnes**
## Description

Adds the .events command that queries the VATSIM api for upcoming events.

If someone is knowledgeable enough to paginate this then go ahead but I have no idea how to...

## Test Results
image

### Intentional url error
image

### CODE REVIEW

File: src/commands/utils/events.ts [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+});
+
+const handleLocaleDateString = (date: Date) => date.toLocaleDateString('en-US', {
+    weekday: 'short',
+    month: 'short',
+    day: 'numeric',
+});
+
+export const events: CommandDefinition = {
+    name: ['events', 'event', 'ev'],
```
Comment (benw202) [inline src/commands/utils/events.ts:22]:
```suggestion
    name: ['events', 'event', 'ev', 'vatevents', 'vatsimevents'],
```

Just some suggestions to make aliases more specific. (Any you choose need adding to command-docs)

**Review by benw202 — COMMENTED** [review]
Can the filename be changed to vatsimEvents? I can see us potentially wanting to use events for FBW events in the future

**Comment by benw202** [comment]
Looking good! 

Are you happy for this to go without having pages? Something we can look at in the future

**Comment by NathanInnes** [comment]
Yup

File: src/commands/utils/vatsimEvents.ts [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+                        name: 'Link',
+                        value: `${link}`,
+                        inline: false,
+                    },
+                ];
+            }).flat();
+
+            const eventsEmbed = makeEmbed({
+                title: 'VATSIM Events',
+                description: 'A list of upcoming vatsimEvents from the VATSIM network.',
```
Comment (benw202) [inline src/commands/utils/vatsimEvents.ts:75]:
```suggestion
                description: 'A list of upcoming events on the VATSIM network.',
```

File: src/commands/utils/vatsimEvents.ts [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+
+const handleLocaleDateString = (date: Date) => date.toLocaleDateString('en-US', {
+    weekday: 'short',
+    month: 'short',
+    day: 'numeric',
+});
+
+export const vatsimEvents: CommandDefinition = {
+    name: ['events', 'event', 'ev', 'vatsimevents', 'vatevents', 'vatev'],
+    description: 'Get the upcoming vatsimEvents from the VATSIM network.',
```
Comment (benw202) [inline src/commands/utils/vatsimEvents.ts:23]:
```suggestion
    description: 'Get the upcoming events on the VATSIM network.',
```

**Review by benw202 — APPROVED** [review]

**Pull request merged**